packages/apps/esm-help-menu-app/src/help-menu/components/styles.scss

```
@@ -6,8 +6,13 @@
   display: flex;
   align-items: center;
   justify-content: space-between;
+  padding: 0.5rem 0.5rem;
```

```suggestion
  padding: layout.$spacing-03;
```

nit: `padding: layout.spacing-03` is shorthand for `padding-top: 0.5rem; padding-right: 0.5rem; padding-bottom: 0.5rem; padding-left: 0.5rem`;

---

Noted.